tsdb/tsm1/file_store.go

```diff
@@ -631,13 +657,23 @@ func (f *FileStore) Open(ctx context.Context) error {
 			continue
 		}
 		f.files = append(f.files, res.r)
+		name := filepath.Base(res.r.Stats().Path)
+		_, seq, err := f.parseFileName(name)
+		if err != nil {
+			return err
+		}
+		if _, ok := counts[seq]; !ok {
+			counts[seq] = 1
+		} else {
+			counts[seq]++
+		}
```

Comment:
Neat trick: only the `++` branch is needed, since maps return the zero value for missing keys. https://play.golang.org/p/NKqO6UFZHn5